blink-search.el

```diff
@@ -90,7 +90,8 @@
                                                     (if load-file-name
                                                         (file-name-directory load-file-name)
                                                       default-directory)))
-(add-to-list 'load-path blink-search-backend-path t)
+(eval-when-compile
+  (add-to-list 'load-path blink-search-backend-path t))
```

Comment:
It's not right, blink-search-backend-path won't add in `load-path` if user not compile blink-search.

Reply:
... I agree this is confusing, and the docstring contradicts the manual: (quote the manual) "This form marks body to be evaluated at compile time but not when the compiled program is loaded." Changed to `eval-and-compile`.